apps/cli/src/commands/sync.ts

```diff
@@ -0,0 +1,70 @@
+import type { Data } from "@webstudio-is/react-sdk";
+import { readFile, writeFile } from "node:fs/promises";
+import { cwd } from "node:process";
+import { join } from "node:path";
+import ora from "ora";
+import { loadProjectDataById } from "@webstudio-is/http-client";
+
+import type { Command } from "../args";
+import { ensureFileInPath, loadGlobalConfigFile } from "../fs-utils";
+import { LOCAL_APP_CONFIG_FILE_NAME, LOCAL_CONFIG_FILE } from "../constants";
+
+export const sync: Command = async (args) => {
+  const spinner = ora("Syncing project data").start();
+  let projectId = args.positionals[1];
```

Comment:
A bit confusing positionals start with 1. Maybe slice command name before pass here?

Reply:
good idea, as we don't need it inside anyways 👍